src/firecracker/swagger/firecracker.yaml

```diff
@@ -1028,6 +1028,9 @@ definitions:
     properties:
       cpu_template:
         $ref: "#/definitions/CpuTemplate"
+      gdb_socket_path:
+        type: string
+        description: Path to the GDB socket. Requires the gdb feature to be enabled.
```

Comment:
Can this break customers that use Swagger to autogenerate the client? I.e. I guess if it's not specified it won't be included in the payload, but I wonder if adding it here is more risky than it's worth.

Reply:
Yeah the scenario I see is, some auto-generator could start including this (optional) field in all request (maybe with value `null`?) and then Firecracker fails because it does not recognize the option.

I think it's farfetched as that would not be a very sane client, but at the same time, I'm not sure there's a lot of value exposing it in the Swagger definition? Then again, a customer could just remove this snippet if it causes issues.

Reply:
yeah, I can see that happening. Ok, maybe we can just add this as a comment  in yaml instead?

Reply:
Yeah, I am good with either solution. Worst case we have the workaround of commenting a few lines when generating the client.

Reply:
I've commented it out for now to be on the safer side